### PR TITLE
feat: add support for session id on TableDataWriteChannel

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

File: google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java [hunk 1]
```diff
@@ -75,10 +78,14 @@ public static final class Builder implements LoadConfiguration.Builder {
...
     private Map<String, String> labels;
     private List<String> decimalTargetTypes;
+    private List<ConnectionProperty> connectionProperties;
+
+    private Boolean createSession;
 
     private Builder() {}
 
     private Builder(WriteChannelConfiguration writeChannelConfiguration) {
+      this();
```
Comment (farhan0102) [inline google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java:88]: this is cool; didn't know about this until today, isn't this a no-op?

File: google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java [hunk 2]
```diff
@@ -274,6 +290,16 @@ public Builder setDecimalTargetTypes(List<String> decimalTargetTypes) {
       return this;
     }
 
+    public Builder setConnectionProperties(List<ConnectionProperty> connectionProperties) {
```
Comment (farhan0102) [inline google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java:293]: not crucial, but if we use lombok, it covers some boiler plate code for getters and setters:
https://projectlombok.org/features/GetterSetter

File: google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java [hunk 3]
```diff
@@ -3733,6 +3769,81 @@ public void testQuerySessionSupport() throws InterruptedException {
...
+    }
+    Job loadJob = bigquery.getJob(jobId);
+    Job completedJob = loadJob.waitFor();
+
+    assertNotNull(completedJob);
+    assertEquals(jobId.getJob(), completedJob.getJobId().getJob());
+    JobStatistics.LoadStatistics statistics = completedJob.getStatistics();
+
+    sessionId = statistics.getSessionInfo().getSessionId();
+    System.out.println(sessionId);
```
Comment (farhan0102) [inline google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java:3807]: im guessing this was used for debugging, and meant to be removed?

**Review by farhan0102 — APPROVED** [review]
minor style stuff as comments you can ignore for now, for the test do we know the file size that was used by the customer that was breaking?

File: google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java (hunk 3 above)
Comment (Neenu1995) [reply google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java]: removed.

**Comment by Neenu1995** [comment]
Do not merge until Part 2 is ready

**Pull request merged**